cmd/fastly/static/config.toml

```diff
@@ -21,42 +21,67 @@ rustup_constraint = ">= 1.23.0"
 [viceroy]
 ttl = "24h"
 
-[[starter-kits.rust]]
-name = "Authenticate at edge with OAuth"
-description = "Connect to an identity provider such as Auth0 using OAuth 2.0 and validate authentication status at the Edge, to authorize access to your edge or origin hosted applications."
-path = "https://github.com/fastly/compute-rust-auth"
-
-[[starter-kits.rust]]
-name = "Beacon termination"
-description = "Capture beacon data from the browser, divert beacon request payloads to a log endpoint, and avoid putting load on your own infrastructure."
-path = "https://github.com/fastly/compute-starter-kit-rust-beacon-termination"
-
-[[starter-kits.rust]]
-name = "Default starter for Rust"
-description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
-path = "https://github.com/fastly/compute-starter-kit-rust-default"
-
-[[starter-kits.rust]]
-name = "Empty starter for Rust"
-description = "An empty starter kit project template."
-path = "https://github.com/fastly/compute-starter-kit-rust-empty"
-
-[[starter-kits.rust]]
-name = "Static content"
-description = "Apply performance, security and usability upgrades to static bucket services such as Google Cloud Storage or AWS S3."
-path = "https://github.com/fastly/compute-starter-kit-rust-static-content"
-
-[[starter-kits.assemblyscript]]
-name = "Default starter for AssemblyScript"
-description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
-path = "https://github.com/fastly/compute-starter-kit-assemblyscript-default"
-
-[[starter-kits.javascript]]
-name = "Default starter for JavaScript"
-description = "A basic starter kit that demonstrates routing, simple synthetic responses and overriding caching rules."
-path = "https://github.com/fastly/compute-starter-kit-javascript-default"
-
-[[starter-kits.javascript]]
-name = "Empty starter for JavaScript"
-description = "An empty application template for the Fastly Compute@Edge environment which simply returns a 200 OK response."
-path = "https://github.com/fastly/compute-starter-kit-javascript-empty"
+[starter-kits]
```

Comment:
What's the benefit in having this section heading?  It looks like all TOML sections have to be fully qualified, so I'm wondering what this does.

Reply:
@triblondon So to clarify, this change was me manually updating the static config for testing purposes. When I come to merge this PR the `static/config.toml` will instead be updated via the `config` Makefile target, which pulls the latest version of the config from the Developer Hub (DevHub). 

This means we need the (internal) DevHub content PR merged first before merging this PR so that the go starter kit will be included in the generated remote config endpoint. Once that's available, then the CLI's CI process will check that the static config in this PR matches it. 

The top level `[starter-kits]` section isn't present in the DevHub's remote CLI config endpoint (simply because we didn't need to be so verbose, it instead only adds the fully qualified version like `[[starter-kits.<LANGUAGE>]]`) but it's worth realising that the TOML parser generates an AST with `starter-kits` as a top-level section and so when the CLI makes changes to the user's _local_ CLI config, the format of the local config matches this more verbose version you see here (that's why you see the verbose formatting here because when developing this branch I manually copied my local config into the static/backup config -- that will disappear once I switch to the `make config` version).